### PR TITLE
Add GeekForGeeks opportunity 

### LINKED ISSUE #1014: Geeks for geeks : Problem of the Day Challenge
## Wild swag opportunity appeared!

- [✓ ] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Geeksforgeeks

Q. Where have you seen the announcement (URL)?  
A. : https://practice.geeksforgeeks.org/problem-of-the-day

Q. What kind of swag (clothing, stickers, ...)?  
A. : Anything of our choice among GFG merchandise, Discount on Gfg courses, Annual GFG premium subscription

Q. Have you received swag from this?  
A. : No

Q. How long did it take?  
A. : Till you complete the challenge

Q. Does it ship worldwide?  
A. : No

- [✓ ] I've checked that the promotion is legit.
- [✓ ] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by FarukhS52**
- [ ] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

adding new swag opportunity ,which was not listed in the dat.json

Fixes #1014

### CODE REVIEW

**Comment by FarukhS52** [comment]
Hey @aslafy-z ,i have added a swag opportunity for coders and developers,can you please review this requested addition and tell me if any changes are required. A feedback would be nice

File: data.json [hunk 1]
```diff
@@ -377,6 +377,15 @@
     "dateAdded": "2018-10-24T11:42:09.000Z",
     "tags": ["clothing", "stickers"]
   },
+  {
+    "name": "GeeksForGeeks",
+    "difficulty": "medium",
+    "description": "Solve daily coding problems and earn coins,and redeem swags",
```
Comment (aslafy-z) [inline data.json:383]: Please refine a bit the description
Add missing space and describe what are the swag.

**Review by aslafy-z — COMMENTED** [review]
I left some comments

**Comment by FarukhS52** [comment]
Hey @aslafy-z  i have made required changes

File: data.json [hunk 2]
```diff
@@ -404,6 +404,15 @@
     "dateAdded": "2018-10-24T11:42:09.000Z",
     "tags": ["clothing", "stickers"]
   },
+  {
+    "name": "GeeksForGeeks",
+    "difficulty": "medium",
+    "description": "Solve daily coding problems and earn coins,and redeem swags such as, T-shirts ,Hoodies and Bags",
```
Comment (aslafy-z) [inline data.json:410]:
```suggestion
    "description": "Solve daily coding problems, earn coins and redeem swags such as t-shirts, hoodies and bags.",
```

File: data.json [hunk 3]
```diff
@@ -404,6 +404,15 @@
     "dateAdded": "2018-10-24T11:42:09.000Z",
     "tags": ["clothing", "stickers"]
   },
+  {
+    "name": "GeeksForGeeks",
+    "difficulty": "medium",
+    "description": "Solve daily coding problems, earn coins and redeem swags such as t-shirts, hoodies and bags.",
+    "reference": "https://practice.geeksforgeeks.org/problem-of-the-day",
+    "image": "https://imgbox.com/LXmCPIG0",
```
Comment (aslafy-z) [inline data.json:412]:
```suggestion
    "image": "https://i.imgur.com/oIAh3jt.png",
```

**Pull request merged**